Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim
```

Comment:
Maybe Python 3.11 just because it's the newest stable version? Or a variable with that as a default.